### ISSUE TITLE
Fix compatibility with Sanic 20.3.0

### ISSUE THREAD

**Issue by brechtm** [pyx/sanic-cookiesession#4]
This change breaks Sanic-CookieSession: https://github.com/huge-success/sanic/pull/1801

**Comment by danielalvsaaker** [comment]
I was about to open a proper pull request, but realized midway that it's not worth it considering the repository is not maintained. I would suggest migrating to [sanic_session](https://github.com/xen/sanic_session) using the in-memory session interface.

**Comment by pyx** [comment]
Sorry about the hiatus.
@danielalvsaaker is you are still interested in using this, could you please reopen the pull request, I will make a new release based on that. Either way, thank you.

**Comment by brechtm** [comment]
> Sorry about the hiatus.

No need to apologize! But a new release with this fix would be highly appreciated.

> @danielalvsaaker is you are still interested in using this, could you please reopen the pull request, I will make a new release based on that. Either way, thank you.

As the owner of this project, you should be able to reopen the PR yourself.

**Comment by pyx** [comment]
@brechtm because PR #5 is broken: hard-coded session object name and inconsistency of object property vs dict element, and the new context type, types.SimpleNamespace does not have dict-like interface, etc. so I actually wrote a fix myself, including updated documentation, code example, and tests.

Then I found out the test client switch from **requests** to **httpx**, and that breaks the tests, e,g `req, resp = app.test_client.get('/inc', cookies=cookies)` does not keep the cookies in the object `resp`, half of the time (heisenbug😂), so I cannot simulate a series of post with states, **httpx** support context manager style code blocks (with statement) and it should work, except `sanic` 's test client does not support this nor expose the underlying `httpx` object.
Before I shoehorn an `httpx` object into tests, I need some reading about the difference between `requests` and `httpx` to find out why it doesn't work, I read the code in `sanic` briefly and it seems okay, passing the `cookies` object to `httpx`, I am feeling like debugging `sanic` and `httpx` at the same time...
I might rewrite the whole test-suit.
